Source/Diagnostics/FlushFormats/FlushFormatCatalyst.cpp

```diff
@@ -0,0 +1,211 @@
+#include "FlushFormatCatalyst.H"
+
+#include "WarpX.H"
+#include "Utils/TextMsg.H"
+#include "Utils/WarpXProfilerWrapper.H"
+
+#include <AMReX.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_REAL.H>
+
+#include <string>
+#include <algorithm>
+
+#ifdef AMREX_USE_CATALYST
+    #include "conduit_cpp_to_c.hpp"
+#endif
```

Comment:
Is this include really used? Do you mean `<conduit.hpp>`?
https://catalyst-in-situ.readthedocs.io/en/latest/for_implementation_developers.html#implementing-catalyst-api